apps/client/src/common/utils/eventsManager.ts

```diff
@@ -23,6 +23,7 @@ export const cloneEvent = (event: OntimeEvent): ClonedEvent => {
     isPublic: event.isPublic,
     skip: event.skip,
     colour: event.colour,
+    currentBlock: event.currentBlock,
```

Comment:
Could we consider renaming it to something like parentBlock, since it hase nothing to to with the actually current block?

Reply:
that is OK with me, lets do it once the groups code is merged so we don't have to deal with so many merge conflicts